build-support/bin/docs_templates/single_option_reference.md.mustache

```diff
@@ -1,16 +1,18 @@
 <div style="color: purple">
```

Comment:
No concrete changes to this, only reformatting of the HTML. It will make it much easier to look over the generated markdown, which I had to do today to debug a 500 error from invalid markdown.